ci-operator/config/openshift/cluster-api-provider-ovirt/openshift-cluster-api-provider-ovirt-master.yaml

```diff
@@ -45,6 +45,10 @@ resources:
       cpu: 100m
       memory: 200Mi
 tests:
+- as: unit
```

Comment:
Is it also possible to execute 
https://github.com/openshift/cluster-api-provider-ovirt/blob/master/hack/fetch-envtest-tools.sh
to fetch `envtest` tools so that I could run the integration tests in a dedicated step as well?